#number of links
uint32 linknum

#D-H coefficients
float64[] ai_1
float64[] di
float64[] alhi_1
float64[] fii
---
#RPY angles
float64[] roll
float64[] pitch
float64[] yaw
float64[] x
float64[] y
float64[] z